packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -7030,12 +7090,48 @@ class Semantics extends SingleChildRenderObjectWidget {
   /// an [ExcludeSemantics] widget and then another [Semantics] widget.
   final bool excludeSemantics;
 
+  /// Whether to block user interactions for the rendering subtree.
+  ///
+  /// Setting this to true will prevent users from interacting with this
+  /// rendering object and its subtree through pointer-related
```

Comment:
"this rendering object": This is not a rendering object. Maybe say "the rendering object configured by this widget"